make.xsh

```diff
@@ -133,30 +133,65 @@ def start_backend_prod():
     """
-    Use the "database-backup" service in the "docker-compose.yml" file to backup
-    all of the tables in the instace of Postgres running locally.
+    Use the "database-backup" service in the "docker-compose.yml" file drop, recreate,
+    and restore all of the tables from S3.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
     $POSTGRES_HOST = get_localhost()
-    docker network prune --force
-    docker-compose run database-backup backup
+    # Checks for a running Postgres container and restores the database if it is running.
+    docker ps | grep postgres && \
+        docker-compose run database-backup restore-from-most-recent || \
+        echo "There is not a Postgres container currently running. Run `make start-database-stack` to start the database."
 
 @makefile.target(tag="run services locally")
-def restore_database():
+def backup_database():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup
+    the database to S3.
     """
-    Use the "database-backup" service in the "docker-compose.yml" file to identify
-    the most recent backup in "infrastructure/containers/postgres/backups" and load that data
-    into the database.
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
+    $POSTGRES_HOST = get_localhost()
+    # Checks if there is a running Postgres container and backs up the database to S3
+    docker ps | grep postgres && \
+        docker-compose run database-backup backup || \
+        echo "There is not a Postgres container currently running. Run `make start-database-stack` to start the database."
 
-    Note that this will wipe the existing data in the database first.
+@makefile.target(tag="run services locally")
+def start_database_stack():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup on a
+    regular inteval specified in the "docker-compose.yml" all of the tables in the
+    databse.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
+    $POSTGRES_HOST = get_localhost()
+    # Runs a postgres instance if one is not already running
     docker network prune --force
-    database_backup_container_id = $(docker ps --quiet --filter name=database-backup)
-    database_backup_container_id = database_backup_container_id.strip()
-    docker exec @(database_backup_container_id) python3 backup_or_restore.py restore-from-most-recent
+    docker swarm init || echo "docker swarm is already initialized :D"
+    docker stack deploy --compose-file docker-compose.yml rootski-database
+    # docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
+    # docker-compose run database-backup backup-on-interval
+
+@makefile.target(tag="run services locally")
+def stop_database_stack():
+    """
+    Tears down the \"rootski-database\" docker-swarm stack and removes
```

Comment:
Hahaha, what's with the `\"`'s here? Did xsh get mad at you for not having them?